inscriptions/ISic003508.xml

```diff
@@ -198,20 +198,21 @@
         <body>
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
-                    <lb n="1"/><g ref="#ivy-leaf"/> <expan><abbr>D</abbr><ex>is</ex></expan> <g ref="#interpunct"/> <expan><abbr>M</abbr><ex>anibus</ex></expan> <g ref="#interpunct"/> <g ref="#ivy-leaf"/>
-                    <lb n="2"/><choice><sic>Belleia</sic><corr>Velleia</corr></choice> <g ref="#interpunct"/> Vesonia <g ref="#interpunct"/> <expan><abbr>vix</abbr><ex>it</ex></expan> <g ref="#interpunct"/>
-                    <lb n="3"/><expan><abbr>ann</abbr><ex>is</ex></expan> <g ref="#interpunct"/> <num value="34">XXXIIII</num> <g ref="#interpunct"/> <expan><abbr>L</abbr><ex>ucius</ex></expan> <g ref="#interpunct"/> Ostrius <g ref="#interpunct"/>
-                    <lb n="4"/>Crescens <g ref="#interpunct"/> uxori <g ref="#interpunct"/> rarissimae
+                    <lb n="1"/><expan><abbr>D</abbr><ex>is</ex></expan> <supplied reason="lost" quantity="1" unit="character"><expan><abbr>M</abbr><ex>anibus</ex></expan></supplied>
```

Comment:
all looks right, except that here and in line 5, we don't attempt to specify the number of characters in a supplied element, since that is already covered by the actual characters recorded; quantification is used only on gap or space, where we are attempting to say how long either the gap or the space is; it's unnecessary on supplied. Simply `<supplied reason="lost">`